src/index.html

```diff
@@ -1,12 +1,396 @@
+            </div>
+            <span class="card__review-text">
+              Reviews: 5
+            </span>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-tag">
+              Price:
+            </div>
+            <div class="card__price-amount">
+              $2,199
+            </div>
+          </div>
+          <a
+            href="#"
+            name="Buy"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            buy
+          </a>
+        </article>
+        <article class="card cards__card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac27inches"
+          >
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">
+            Product code: 195434
+          </div>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <span class="card__review-text">
+              Reviews: 5
+            </span>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-tag">
+              Price:
+            </div>
+            <div class="card__price-amount">
+              $2,199
+            </div>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
+            buy
+          </a>
+        </article>
+        <article class="card cards__card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac27inches"
+          >
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">
+            Product code: 195434
+          </div>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <span class="card__review-text">
+              Reviews: 5
+            </span>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-tag">
+              Price:
+            </div>
+            <div class="card__price-amount">
+              $2,199
+            </div>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
```

Comment:
You forgot to add the qa attribute for tests

Reply:
I did but only for the first link, from the first card. :D